src/utils/tableRequestStatusIndex.ts

```diff
@@ -1,78 +1,90 @@
- * Get a hash table of status enum strings and their status order.
+ * Hash tables of status enum strings and their status order.
  * Used for the request table status column sort compare fn.
  */
 // Unfortunately the order of original enums from `schema.graphql` are lost
 // when types are generated for the frontend in `global-types`.
 // Generated enum strings are maintained in such a way here for react table column sorting.
-export function parseSortIndex(arr: string[]): () => { [v: string]: number } {
-  return once(() => Object.fromEntries(arr.map((v, i) => [v, i])));
+/*
+Produce sort index objects with something like this:
+
+  function parseSortIndex(arr) {
+    return Object.fromEntries(arr.map((v, i) => [v, i]));
+  }
+
+Input
+[
+ SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_NEW,
+ SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_IN_PROGRESS,
+ SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_SUBMITTED,
+]
+
+Output
+{
+  INITIAL_REQUEST_FORM_NEW: 0,
+  INITIAL_REQUEST_FORM_IN_PROGRESS: 1,
+  INITIAL_REQUEST_FORM_SUBMITTED: 2,
 }
 
-export const SystemIntakeStatusRequesterIndex = parseSortIndex([
-  SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_NEW,
-  SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_IN_PROGRESS,
-  SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_SUBMITTED,
-  SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_EDITS_REQUESTED,
-  SystemIntakeStatusRequester.DRAFT_BUSINESS_CASE_IN_PROGRESS,
-  SystemIntakeStatusRequester.DRAFT_BUSINESS_CASE_SUBMITTED,
-  SystemIntakeStatusRequester.DRAFT_BUSINESS_CASE_EDITS_REQUESTED,
-  SystemIntakeStatusRequester.GRT_MEETING_READY,
-  SystemIntakeStatusRequester.GRT_MEETING_AWAITING_DECISION,
-  SystemIntakeStatusRequester.FINAL_BUSINESS_CASE_IN_PROGRESS,
-  SystemIntakeStatusRequester.FINAL_BUSINESS_CASE_SUBMITTED,
-  SystemIntakeStatusRequester.FINAL_BUSINESS_CASE_EDITS_REQUESTED,
-  SystemIntakeStatusRequester.GRB_MEETING_READY,
-  SystemIntakeStatusRequester.GRB_MEETING_AWAITING_DECISION,
-  SystemIntakeStatusRequester.LCID_ISSUED,
-  SystemIntakeStatusRequester.LCID_EXPIRED,
-  SystemIntakeStatusRequester.LCID_RETIRED,
-  SystemIntakeStatusRequester.NOT_GOVERNANCE,
-  SystemIntakeStatusRequester.NOT_APPROVED,
-  SystemIntakeStatusRequester.CLOSED
-]);
+*/
+
+export const SystemIntakeStatusRequesterIndex: Record<string, number> = {
+  INITIAL_REQUEST_FORM_NEW: 0,
+  INITIAL_REQUEST_FORM_IN_PROGRESS: 1,
+  INITIAL_REQUEST_FORM_SUBMITTED: 2,
+  INITIAL_REQUEST_FORM_EDITS_REQUESTED: 3,
+  DRAFT_BUSINESS_CASE_IN_PROGRESS: 4,
+  DRAFT_BUSINESS_CASE_SUBMITTED: 5,
+  DRAFT_BUSINESS_CASE_EDITS_REQUESTED: 6,
+  GRT_MEETING_READY: 7,
+  GRT_MEETING_AWAITING_DECISION: 8,
+  FINAL_BUSINESS_CASE_IN_PROGRESS: 9,
+  FINAL_BUSINESS_CASE_SUBMITTED: 10,
+  FINAL_BUSINESS_CASE_EDITS_REQUESTED: 11,
+  GRB_MEETING_READY: 12,
+  GRB_MEETING_AWAITING_DECISION: 13,
+  LCID_ISSUED: 14,
+  LCID_EXPIRED: 15,
+  LCID_RETIRED: 16,
+  NOT_GOVERNANCE: 17,
+  NOT_APPROVED: 18,
+  CLOSED: 19
+} as const;
```

Comment:
Instead of typing the string as the key, you should use the enum 

```
export const SystemIntakeStatusRequesterIndex: Record<SystemIntakeStatusRequester, number> = {
[SystemIntakeStatusRequester.INITIAL_REQUEST_FORM_NEW]: 0
```